### PR TITLE
feature/40: added room coolers screen

### PR DESCRIPTION
**Pull request by alena-andr**

### CODE REVIEW

File: src/screens/Coolers/Coolers.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React, {ReactText, PureComponent} from 'react';
+import {ScrollView} from 'react-native';
+import {NavigationScreenProps} from 'react-navigation';
+
+import CoolerItem from './components/CoolerItem/CoolerItem';
+
+import styles from './coolersStyles';
+
+const mock = [{
```
Comment (CodeAnnihilator) [inline src/screens/Coolers/Coolers.tsx]: move to seed

File: src/screens/Coolers/components/CoolerItem/CoolerItem.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		series: string;
+		status: string;
+	};
+	onPress: (id: ReactText) => void;
+}
+
+export default class CoolerItem extends PureComponent<ICoolerItemProps> {
+	public render() {
+		const {
+			cooler: {
```
Comment (CodeAnnihilator) [inline src/screens/Coolers/components/CoolerItem/CoolerItem.tsx]: don't use deep destructuring

File: src/screens/Coolers/components/CoolerItem/CoolerItem.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+				id,
+				power,
+				title,
+				series,
+				status,
+			},
+			onPress,
+		} = this.props;
+		const statusColor = (status === 'active' ?
+			colors['$green600'] : status === 'off' ?
```
Comment (CodeAnnihilator) [inline src/screens/Coolers/components/CoolerItem/CoolerItem.tsx:26]: add colors in global constants

Comment (alena-andr) [reply src/screens/Coolers/components/CoolerItem/CoolerItem.tsx:26]: discussed

File: src/screens/Coolers/components/CoolerItem/CoolerItem.tsx (hunk 2 above)
Comment (alena-andr) [reply src/screens/Coolers/components/CoolerItem/CoolerItem.tsx]: done

File: src/screens/Coolers/Coolers.tsx (hunk 1 above)
Comment (alena-andr) [reply src/screens/Coolers/Coolers.tsx]: done

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**